sys/posix/Kconfig

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menu "Posix"
+    depends on TEST_KCONFIG
+
+config MODULE_POSIX_HEADERS
```

Comment:
Should this be outside the menu?

Reply:
I think it is ok here, so it inherits the `TEST_KCONFIG` dependency.